src/main.js

```diff
@@ -34,3 +36,4 @@ const unsubscribe = firebase.auth()
   })
   unsubscribe()
 })
+export const db = app.firestore()
```

Comment:
change the name of this variable to what is in master, and remember to make the change where ever it is referenced